dpnp/dpnp_iface_arraycreation.py

```diff
@@ -552,7 +552,7 @@ def empty_like(x1,
 
     """
 
-    if not isinstance(x1, dpnp.ndarray):
+    if not dpnp.isarray(x1):
```

Comment:
It needs to update limitations block, since `x1` as :class:`dpctl.tensor.usm_ndarray` is also allowed